geopandas/geoseries.py

```diff
@@ -749,13 +749,24 @@ def notnull(self):
         return self.notna()
 
     def fillna(self, value=None, method=None, inplace=False, **kwargs):
-        """Fill NA values with a geometry (empty polygon by default).
+        """
+        Fill NA values with geometry (or geometries).
+
+        ``method`` is currently not implemented for pandas <= 0.12.
+
+        Parameters
+        ----------
+        value : Geometry or GeoSeries, default None
+            - ``None`` : Fill empty polygon to all missing values.
+            - ``Geometry`` : Fill it to all missing values.
+            - ``GeoSeries`` : It requires having the same length as 'self'.
```

Comment:
```suggestion
        value : Geometry or GeoSeries, default None
            If None is passed, NA values will be filled with GEOMETRYCOLLECTION EMPTY. 
            If ``pygeos.Geometry`` or a shapely geometry object is passed, it will be used to fill all missing values. 
            If a ``GeoSeries`` or ``GeometryArray`` are passed, missing values will be filled based on the corresponding index locations.
```
(the line wrapping on this will need fixing)